src/FlexVotingDelegatable.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {Context} from "@openzeppelin/contracts/utils/Context.sol";
+import {Checkpoints} from "@openzeppelin/contracts/utils/structs/Checkpoints.sol";
+
+import {FlexVotingBase} from "src/FlexVotingBase.sol";
+
+/// @notice This is an abstract contract designed to make it easy to build
+/// clients for governance systems that inherit from GovernorCountingFractional,
+/// a.k.a. Flexible Voting governors.
+///
+/// This contract extends FlexVotingBase, adding the ability to subdelegate one's
+/// internal voting weight. It is meant to be inherited from in conjunction with
+/// FlexVotingClient. Doing so makes the following usecase possible:
```

Comment:
I know we discussed this before, so can you remind me: If this contract is meant to be used with `FlexVotingClient`, then why not inherit from `FlexVotingClient` directly, instead of from the base? Why make the user figure out to inherit from both.

Reply:
Ah, I found where we talked about this:

<img width="890" alt="image" src="https://github.com/user-attachments/assets/764b151e-8cac-4c80-b9b9-82a8e2d13a14" />

It is more modular, but it's modularity that's not really interesting, since there really isn't a usecase for `FlexVotingBase + FlexVotingDelegatable` without also including `FlexVotingClient`.

I do still kind of agree that it is more self-documenting to have `MyContract is FlexVotingClient, FlexVotingDelegatable` than it is to just do `MyContract is FlexVotingDelegatable`.

What do you think is more important in this case: simplicity (one contract to inherit from) or self-documentation?